src/image-helpers.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @class ImageHelpers
+ */
+export default class ImageHelpers {
+
+  /**
+   * Generates the image src for a resized image with maximum dimensions `maxWidth` and `maxHeight`.
+   * Images do not scale up.
+   *
+   * @method imageForSize
+   * @param {Object} image the original image object to generate the image src for
+   * @param {Integer} maxHeight the maximum height for the image
+   * @param {Integer} maxWidth the maximum width for the image
+   * @return {String} The image src for the resized image
+   */
+  imageForSize(image, maxHeight, maxWidth) {
+    const tokens = image.src.split('.');
+
+    // Take the second last token and append the dimensions
+    tokens[tokens.length - 2] = tokens[tokens.length - 2].concat(`_${maxHeight}x${maxWidth}`);
```

Comment:
I think it may be safer to split on `/` (sorry). `/` has specific meaning in the URL, and likely won't be added arbitrarily. `.` could easily be added in URL params for concerns we're not aware of.